doc/distrib/NodeHelpFiles/_review/Revit.Elements.Viewport.LabelLineLength.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Viewport.LabelLineLength` gets the length of the viewport line, in paper space units.
+
+In the example below, all of the viewports from the selected sheet are collected and the line lengths are reported.
```

Comment:
For clarity, can be rephrased as "In the example below, all of the viewports from the selected sheet are collected and the lengths of the viewport label lines are retrieved."